astropy/io/misc/pickle_helpers.py

```diff
@@ -72,7 +80,16 @@ def fnunpickle(fileorname, number=0, usecPickle=True):
     return res
 
 
-def fnpickle(object, fileorname, usecPickle=True, protocol=None, append=False):
+def fnpickle(object, fileorname, **kwargs):
```

Comment:
okay, here it makes sense to use `**kwargs`, but probably worth mentioning that `protocol` and `append` now are keyword-only in the changelog?